src/amuse/__init__.py

```diff
@@ -27,12 +27,34 @@
 """
 import numpy
 
+from amuse.io import read_set_from_file, write_set_to_file
+from amuse.units import (
+    units, nbody_system, constants, generic_unit_system,
+)
+from amuse.units.quantities import (
+    ScalarQuantity, VectorQuantity, AdaptingVectorQuantity,
+    zero
+)
+from amuse.datamodel import (
+    Particle, Particles, ParticlesSuperset, Grid,
+    particle_attributes,
```

Comment:
why particle_attributes? In general I don't think
```
from amuse.datamodel import Particles
```
etc is so bad (a lot of submodules already imported into datamodel..).. same goes for amuse.io

Reply:
mainly because I was copying from lab.py... particle_attributes probably shouldn't be imported here (I can't recall using it).

you're right, it's not so bad - but still, having the "most essential" bits available in the amuse namespace can help new users (and hopefully prevent bad style imports).